tests/e2e/e2e_test.go

```diff
@@ -41,67 +42,98 @@ func (s *IntegrationTestSuite) Test01IBCTokenTransfer() {
 // This is the reason for prefixing the name with 02 to ensure
 // correct order.
 func (s *IntegrationTestSuite) Test02CreatePool() {
-	chain := s.configurer.GetChainConfig(0)
-	node, err := chain.GetDefaultNode()
+	chainA := s.configurer.GetChainConfig(0)
+	chainB := s.configurer.GetChainConfig(1)
```

Comment:
Why do we need to run this on both chains? If it works on one chain, it should work on another chain as well since they are identical. I think duplication only adds to time and increases complexity

Reply:
Discussed this offline, but for anyone else reading, I was using the ordered tests as state creation. After a discussion with Roman, we decided we will try to decouple as many of these tests as possible in order to eventually allow most of these tests to run independently.